qt/widgets/instrumentview/test/InstrumentWidget/InstrumentWidgetTest.h

```diff
@@ -64,178 +64,213 @@ class InstrumentWidgetTest : public CxxTest::TestSuite {
   }
 
   void test_constructor() {
```

Comment:
So I had a thought about how to do this more gracefully. For this PR let's leave it as-is, but could we do a follow up to `main` with something like:

``` c++
class InstrumentWidgetTest {
public:
    InstrumentWidgetTest(bool useThreads);

    void test_constructor() {
  void test_constructor() {
      auto qtMock = makeQtDisplay();
      auto glMock = makeGL();


    // Move m_useThreads out of args into construct
    auto instance = construct("test_ws", makeDisplay(), qtMock.get(), glMock.get(), 22);
...etc.
}

private:
   bool m_useThreads;
```

Then simply build two test cases like this:

```c++
class InstrumentWidgetTestThreaded : public InstrumentWidgetTest, public CxxTest::TestSuite {
public:
    InstrumentWidgetTestThreaded() : InstrumentWidgetTest(true) {}
}

class InstrumentWidgetTestSingleThread : public InstrumentWidgetTest, public CxxTest::TestSuite {
public:
    InstrumentWidgetTestThreaded() : InstrumentWidgetTest(false) {}
}
```